trendy-fashion/src/Components/Filter.jsx

```diff
@@ -1,365 +0,0 @@
-import React, { useEffect, useState } from 'react'
-import {
-  Accordion,
-  AccordionItem,
-  AccordionButton,
-  AccordionPanel,
-  AccordionIcon,
-  Box
-} from '@chakra-ui/react'
-import SeprateCategory from './SeprateCategory'
-import { useSearchParams, useParams, useLocation } from 'react-router-dom';
-import { getBrandData } from "../utils";
-const Filter = () => {
-  const { param } = useParams();
-  const location = useLocation();
-  const [searchParams, setSearchParams] = useSearchParams();
-  const initialCategory = searchParams.getAll('brandName');
-  const initSort = searchParams.getAll('sort');
-  const [sortBy, setSortBy] = useState(initSort || "")
-  const [category, setCategory] = useState(initialCategory || []);
-  const [discountLte, setDiscountLte] = useState([]);
-  const [discountGte, setDiscountGte] = useState([]);
-  const [priceLte, setPriceLte] = useState([]);
-  const [priceGte, setPriceGte] = useState([]);
-
-  const handleFilterCheckbox = (e) => {
-    e.preventDefault();
-    const newCategories = [...category];
-    if (newCategories.includes(e.target.value)) {
-      newCategories.splice(newCategories.indexOf(e.target.value), 1)
-    }
-    else {
-      newCategories.push(e.target.value);
-    }
-    setCategory(newCategories);
-  }
-
-  const discountChangeHandler = (e) => {
-    const limits = e.target.value.split("-");
-    const lower = limits[0];
-    const higher = limits[1];
-    let lte = [...discountLte];
-    if (lte.includes(higher)) {
-      let index = lte.indexOf(higher);
-      lte.splice(index, 1);
-    }
-    else {
-      lte.push(higher);
-    }
-
-    let gte = [...discountGte];
```

Comment:
What are lte and gte? Give meaningful names to your variables and also try to add comments to explain things.